build/python/backend/Dockerfile

```diff
@@ -94,13 +88,21 @@ RUN apt-get -qq update && \
 
 # Install JTR
 RUN apt-get -qq update \
-  && apt-get install -qq --no-install-recommends -y git python build-essential ca-certificates libssl-dev zlib1g-dev yasm libgmp-dev libpcap-dev libbz2-dev libgomp1
+  && apt-get install -qq --no-install-recommends -y  \
+    ca-certificates \
+    libssl-dev \
+    zlib1g-dev \
+    yasm \
+    libgmp-dev \
+    libpcap-dev \
+    libbz2-dev \
+    libgomp1
 # The maintainer isn't big on releases or tags so grab an arbitrary, but consistent, commit.
 # Additionally jump through some extra hoops to get the single commit to save some download time.
 RUN mkdir jtr && cd jtr && git init && git remote add origin https://github.com/openwall/john.git && git fetch --depth 1 origin b5c10480f56ff1b5d76c6cbdaf9c817582ee2228 && git reset --hard FETCH_HEAD \
   && rm -rf /jtr/.git \
   && cd /jtr/src \
-  && ./configure --disable-native-tests \
+  && ./configure \
```

Comment:
Is there a reason to get rid of this?

https://github.com/sublime-security/strelka/pull/21

I don't think we should unless we've confirmed we won't run into the issues this fixed.